ch01.adoc

```diff
@@ -228,3 +228,14 @@ Briefly the new attributes allow:
 * Description of climatological statistics.
 * Data compression for variables with missing values.
 
+
+[[ugrid-conventions, Section 1.6, "UGRID Conventions"]]
+=== UGRID Conventions
+
+These conventions implicitly incorporate parts of the UGRID conventions for storing unstructured (or flexible mesh) data in netCDF files using mesh topologies <<UGRID>>.
+Only version 1.0 of the UGRID conventions is allowed.
+The UGRID conventions description is referenced from, rather than rewritten into, this document and the canonical description of how to store mesh topologies is only to be found at <<UGRID>>.
+A summary indicating how UGRID relates to other parts of the CF conventions, and which features of UGRID are excluded from CF, can be found in <<mesh-topology-variables>>.
+To reduce the chance of ambiguities arising from their accidental re-use, all of the UGRID standardized attributes are specified in <<appendix-mesh-topology-attributes>> and <<attribute-appendix>>.
+
+The UGRID conventions have their own conformance document, which should be used in conjunction with the CF conformance document when checking the validity of datasets.
```

Comment:
The conformance document has been updated: https://github.com/cf-convention/cf-conventions/pull/459/files#diff-3b9c470edad8a09f463987db632803f1ecc22561199fa5771745ad472a62e0ee - which just devolves conformance to the UGRID conformance document.

Reply:
> to have some summary of how the data model has changed,

I'm not in favour of writing into the conventions how they are different to a certain previous versions. That is the role of the issue discussion, I think.

Reply:
> The conformance document has been updated

Aha, sorry I clearly completely missed that when working through the PR :slightly_smiling_face: Devolving sounds wise. All good!

Reply:
RE:

> I'm not in favour of writing into the conventions how they are different to a certain previous versions. That is the role of the issue discussion, I think.

Fair enough, that seems sensible - but for the record and to check my understanding, since we are linked to the Issue here - is it indeed what I understand from looking in detail, that:

> all that has really changed is that we have the new constructs 'DomainTopology' and 'CellConnectivity'

?